edx_sailthru/sailthru_content/services/sailthru_translation_service.py

```diff
@@ -46,7 +46,8 @@ def _create_course_vars(self, course, course_run, url, site_name, program_dictio
             'course_run': True,
             'marketing_url': url,
             'course_id': course['key'],
-            'course_run_id': course_run['key']
+            'course_run_id': course_run['key'],
+            'marketable': 1,  # Mark the course_run to be marketable when updated through this script
```

Comment:
@clintonb I talked with Jeanie and she agreed we should stick with "0" and "1" values. I'll make the update to "course_run" variable as well to use "0" and "1" value.